extensions/amp-story/1.0/media-pool.js

```diff
@@ -850,6 +850,14 @@ export class MediaPool {
       return Promise.resolve();
     }
 
+    // When a video is muted, reset its volume to the default value of 1.
+    if (mediaType == MediaType.VIDEO) {
+      let parent = domMediaEl.parentElement;
+      if (parent) {
```

Comment:
I don't think this `if` statement is necessary, we can reset the volume regardless